website/docs/automation/gitops/argocd/ui.md

```diff
@@ -1,12 +1,14 @@
 ---
-title: 'Server bootstrap'
+title: 'Accessing Argo CD'
 sidebar_position: 10
 weight: 10
 ---
- 
-For the purpose of this lab, Argo CD server has already been exposed outside of the cluster using Load Balancer type.
 
-Get the URL from Argo CD service resource:
+:::info
+For the purpose of this lab, Argo CD server has already been exposed outside of the cluster using `Load Balancer` type. To see how we set up Argo CD in the cluster with EKS Terraform Blueprint, please refer to this [section](https://github.com/aws-samples/eks-workshop-v2/blob/d12e0008d688460f2d0a93c83e1ff3c5b03f72e6/terraform/modules/cluster/addons.tf#L240-L254).
+::: 
```

Comment:
The URL needs to be updated later to reflect the correct files and lines number.